cmd/cosign/cli/options/attest.go

```diff
@@ -64,6 +65,10 @@ func (o *AttestOptions) AddFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVarP(&o.Recursive, "recursive", "r", false,
 		"if a multi-arch image is specified, additionally sign each discrete image")
 
+
+	cmd.Flags().BoolVarP(&o.Replace, "replace", "", false,
```

Comment:
Add a string describing this flag's behavior.

Also, `BoolVarP` is to enable a short flag name (like `--recursive` and -r` above). If this one doesn't have a short flag name, it can just be `BoolVar` and drop the `""`.

Reply:
thank you @imjasonh, done.

Reply:
The `replace` option still had no description in the current main branch and I have a really hard time finding out what exactly it does.